com.unity.multiplayer.mlapi/Runtime/Messaging/InternalMessageSender.cs

```diff
@@ -10,69 +10,76 @@
 
 namespace MLAPI.Messaging
 {
-    internal static class InternalMessageSender
+    internal class InternalMessageSender
```

Comment:
We could leave this class static (and stateless) and just pass in NM in each send.  What does the room think?